lib/paper_trail/sinatra.rb

```diff
@@ -27,13 +31,31 @@ def user_for_paper_trail
       current_user
     end
 
+    def info_for_paper_trail
+      {}
+    end
+
+    def paper_trail_enabled_for_request
+      ::PaperTrail.enabled?
+    end
```

Comment:
Please add documentation for both of these methods. You can copy the documentation from `paper_trail/frameworks/rails/controller.rb`